src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -464,85 +465,92 @@ protected internal virtual DurableOrchestrationClient GetClient(OrchestrationCli
             };
         }
 
-        internal void RegisterOrchestrator(FunctionName orchestratorFunction, OrchestratorInfo orchestratorInfo)
+        internal void RegisterOrchestrator(FunctionName orchestratorFunction, RegisteredFunctionInfo orchestratorInfo)

```

Comment:
Should we consider verifying that `orchestratorInfo.IsDerigistered=false`?

Reply:
I guess I can just explicitly set it to `false` here.